src/server.ts

```diff
@@ -5,28 +5,67 @@ import { Client } from "pg";
 import { getEnvVarOrFail } from "./support/envVarUtils";
 import { setupDBClientConfig } from "./support/setupDBClientConfig";
 
-dotenv.config(); //Read .env file lines as though they were env vars.
+dotenv.config();
 
 const dbClientConfig = setupDBClientConfig();
 const client = new Client(dbClientConfig);
 
-//Configure express routes
 const app = express();
 
-app.use(express.json()); //add JSON body parser to each following route handler
-app.use(cors()); //add CORS support to each following route handler
+app.use(express.json());
+app.use(cors());
 
 app.get("/", async (_req, res) => {
-    res.json({ msg: "Hello! There's nothing interesting for GET /" });
+    res.json({ msg: "There's nothing interesting for GET /" });
 });
 
-app.get("/health-check", async (_req, res) => {
+app.get("/pastes", async (_req, res) => {
     try {
-        //For this to be successful, must connect to db
-        await client.query("select now()");
-        res.status(200).send("system ok");
+        const text = "select * from pastes";
+        const response = await client.query(text);
+        res.status(200).json(response.rows);
+    } catch (error) {
+        console.error(error);
+        res.status(500).send("An error occurred. Check server logs.");
+    }
+});
+
+app.get("/pastes/:id", async (req, res) => {
```

Comment:
it doesn't look like this function is being used? if not then it should be removed from the main branch/ code base